src/main/java/models/content/AnotherContent.java

```diff
@@ -0,0 +1,26 @@
+package models.content;
+
+import models.games.abstracts.AbstractContent;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class AnotherContent extends AbstractContent {
+    public AnotherContent() {
+        List<String> pair = new ArrayList<>();
+        pair.add("No content!");
+        pair.add("No content!");
+        getAnswers().put(0,pair);
```

Comment:
formatting issue: should be a space after comma

Reply:
- [x] added space after comma